agent/component/fillup.py

```
@@ -76,7 +76,13 @@ def _resolve_input_value(self, value, layout_recognize):
             return FileService.get_files(files, layout_recognize=layout_recognize)
 
         if isinstance(value, dict):
-            return value.get("value")
+            raw = value.get("value")
+            if value.get("type") == "object" and isinstance(raw, str) and raw.strip():
+                try:
+                    return json.loads(raw)
+                except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)